performance_manager/lib/l1_full_trip_events.py

```diff
@@ -308,7 +308,7 @@ def update_with_new_events(
     alternate approach is used here where just hashes are selected that represent
     update records and each column is updated seperatly from insert_dataframe
     """
-    process_logger = ProcessLogger("l1_update_events")
+    process_logger = ProcessLogger("l1_load_table")
```

Comment:
Minor logging changes in `performance_manager` files. This is a work in a progres.